### DISCUSSION TITLE
A lot of views rendered slows down the pagination rendering

### DISCUSSION THREAD

**Discussion by man0l** [filamentphp/filament#9422] [Help]
How can I improve this part of the filament admin?

![image](https://github.com/filamentphp/filament/assets/1413216/3527b04e-1a09-4a40-9929-91185ae9ffcf)

**Comment by sinnbeck** [comment]
You can try caching views
```
php artisan view:cache
```
But are you sure its due to the views?

**Reply by VincvanAs** [reply]
Tbh 39 models does not look efficient

**Reply by sinnbeck** [reply]
Yeah or it could badly optimized queries. Hard to tell
